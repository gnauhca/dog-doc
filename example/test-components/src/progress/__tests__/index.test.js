import React from 'react';
import { mount } from 'enzyme';

describe('Progress', () => {
  it('base demo works fine', () => {
    const Demo = require('../demos/base').default;
    const wrapper = mount(<Demo />);
    expect(wrapper).toMatchSnapshot();
  });
  it('circle demo works fine', () => {
    const Demo = require('../demos/circle').default;
    const wrapper = mount(<Demo />);
    expect(wrapper).toMatchSnapshot();
  });
});
