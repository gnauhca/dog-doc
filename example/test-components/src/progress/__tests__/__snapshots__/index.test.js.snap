// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress base demo works fine 1`] = `
<_default>
  <div>
    <Progress
      inline={true}
      size="default"
      type="bar"
      value={0.6}
      valueText={[Function]}
    >
      <div
        className="ten-progress ten-progress--size-default ten-progress--type-bar ten-progress--inline"
      >
        <div>
          <div
            className="ten-progress__bar"
          >
            <div
              className="ten-progress__bar-inner"
              style={
                Object {
                  "width": "60%",
                }
              }
            />
          </div>
          <span
            className="ten-progress__valuetext"
          >
            60%
          </span>
        </div>
      </div>
    </Progress>
    <Progress
      inline={true}
      size="large"
      type="bar"
      value={0.7}
      valueText={[Function]}
    >
      <div
        className="ten-progress ten-progress--size-large ten-progress--type-bar ten-progress--inline"
      >
        <div>
          <div
            className="ten-progress__bar"
          >
            <div
              className="ten-progress__bar-inner"
              style={
                Object {
                  "width": "70%",
                }
              }
            />
          </div>
          <span
            className="ten-progress__valuetext"
          >
            70%
          </span>
        </div>
      </div>
    </Progress>
  </div>
</_default>
`;

exports[`Progress circle demo works fine 1`] = `
<_default>
  <div>
    <Progress
      inline={true}
      size="default"
      type="circle"
      value={0.6}
      valueText={[Function]}
    >
      <div
        className="ten-progress ten-progress--size-default ten-progress--type-circle ten-progress--inline"
      >
        <div>
          <svg
            viewBox="0 0 146 146"
          >
            <path
              className="ten-progress__path-bg"
              d="
    M 73 4 
    A 69 69 0 0 1 73 142
    A 69 69 0 0 1 73 4
  "
              fill="none"
              strokeWidth={8}
            />
            <path
              className="ten-progress__path-value"
              d="
    M 73 4 
    A 69 69 0 0 1 73 142
    A 69 69 0 0 1 73 4
  "
              fill="none"
              strokeDasharray="433.54 433.54"
              strokeDashoffset={173.41600000000003}
              strokeWidth={8}
            />
          </svg>
          <span
            className="ten-progress__valuetext"
          >
            60%
          </span>
        </div>
      </div>
    </Progress>
    <Progress
      inline={true}
      size="large"
      type="circle"
      value={1}
      valueText={[Function]}
    >
      <div
        className="ten-progress ten-progress--size-large ten-progress--type-circle ten-progress--inline"
      >
        <div>
          <svg
            viewBox="0 0 146 146"
          >
            <path
              className="ten-progress__path-bg"
              d="
    M 73 4 
    A 69 69 0 0 1 73 142
    A 69 69 0 0 1 73 4
  "
              fill="none"
              strokeWidth={8}
            />
            <path
              className="ten-progress__path-value"
              d="
    M 73 4 
    A 69 69 0 0 1 73 142
    A 69 69 0 0 1 73 4
  "
              fill="none"
              strokeDasharray="433.54 433.54"
              strokeDashoffset={0}
              strokeWidth={8}
            />
          </svg>
          <span
            className="ten-progress__valuetext"
          >
            100%
          </span>
        </div>
      </div>
    </Progress>
  </div>
</_default>
`;
